app/containers/WalletManager/Wallet.jsx

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React from 'react'
+
+import TextInput from '../../components/Inputs/TextInput'
+import Button from '../../components/Button'
+import Delete from '../../assets/icons/delete.svg'
+import Edit from '../../assets/icons/edit.svg'
+import Confirm from '../../assets/icons/confirm.svg'
+import Close from '../../assets/icons/close.svg'
+import styles from './WalletManager.scss'
+
+type Props = {
+  handleDelete: Function,
```

Comment:
Its best to describe the signature of the function.
`(e: SyntheticEvent<*>) => any` (something like that)